config/software/berkshelf.rb

```diff
@@ -43,4 +43,20 @@
 
   gem "install pkg/berkshelf-*.gem" \
       " --no-ri --no-rdoc", env: env
+
+  if windows?
+    block "Clean up large object files" do
+      lib_path = File.join(
+        "#{install_dir.gsub(/\\/, '/')}",
+        "embedded/lib/ruby/gems/**/gems",
```

Comment:
if i'm reading this correctly it can be nuked on every platform as part of general cleanup since its just the gem cache, it has nothing to do specifically with depselector or these software configs